light_curves/requirements_light_curve_generator.txt

```
@@ -14,7 +14,7 @@ astroquery>=0.4.8.dev0
 acstools
 lightkurve
 alerce
-lsdb>=0.4.5
+lsdb>=0.5.1
```

```suggestion
lsdb>=0.5.2
```

Looks like using a HATS Collection requires 0.5.2 (for ZTF).

---

done.